colp_build/sql/name.sql

```diff
@@ -9,104 +9,110 @@ SELECT
-		                                                    ),
-		                                                'F\.H\.A\.\.',
-		                                                'FHA '
-		                                                ),
-		                                            'GDN\s|GARD\s',
-		                                            'GARDEN '
-		                                            ),
-	                                            'AVE\)|AVE\.\)',
-	                                            'AVENUE)'
-	                                            ),
-                                            '\(AVE|\(AVE\.',
-                                            '(AVENUE'
-                                            ),
-                                        '\sAVE\s|\sAVE\.\s|\sAVE-\s|\sAVE\.\s|\sAV\s',
-                                        ' AVENUE '
-                                        ),
-                                    'HUNTS%POINT%AVENUE%REHAB',
-                                    'HUNTS POINT AVENUE REHAB'
-                                    ),
-                                'AUTH\s',
-                                'AUTHORITY '
-                                ),
-                            'BKLN|BK|BKLYN',
-                            'BROOKLYN'
-                            ),
-                        '\sBX\s|\sBRNX\s|/BX\s|\sBX\.\s',
-                        ' BRONX '
-                        ),
-                    'QNS',
-                    'QUEENS'
-                    ),
-                '\sSI\s',
-                ' STATEN ISLAND '
-                ),
-            'MN\s|MAHN\s',
-            'MANHATTAN '
-            ),
-    'REHABS\(|REHAB\(',
-    'REHAB ('
-    ),
-'REHABS|REHAS',
-'REHAB'
-)) as new_name,
+			REGEXP_REPLACE(
+				REGEXP_REPLACE(
+					REGEXP_REPLACE(
+						REGEXP_REPLACE(
+							REGEXP_REPLACE(
+								REGEXP_REPLACE(
+									REGEXP_REPLACE(
+										REGEXP_REPLACE(
+											REGEXP_REPLACE(
+												REGEXP_REPLACE(
+													REGEXP_REPLACE(
+														REGEXP_REPLACE(
+															REGEXP_REPLACE(
+																REGEXP_REPLACE(
+																	REGEXP_REPLACE(
+																		REGEXP_REPLACE(
+																			REGEXP_REPLACE(
+																				REGEXP_REPLACE(
+																					REGEXP_REPLACE(
+																						REGEXP_REPLACE(
+																							REGEXP_REPLACE(
+																								REGEXP_REPLACE(
+																									REGEXP_REPLACE(
+																										REGEXP_REPLACE(
+																											REGEXP_REPLACE(' '||TRIM(parcel_name)||' ', 
+																											'\s&\s|&', 
+																											' AND '
+																											), 
+																										'PLAYLOT\s|PLGD\s|PLAYGRD\s|PLGRD\s|PLAYGND\s|PLYGRND/|PLYGRND\s|PLYGD\s|PLYGD|PLAYGRND\s|PLYGRD\s|PLYGRD/|PLGRD/|PLGRD\s|PLGD.\s|PLGD\s|PLYGD/|PLYGD|PLYGD\s|PLAYGD\s|PLGND\s|PLGD/|PLGD-|PLAYGND-', 
```

Comment:
Note: it'd be useful to have you explain why some phrases that are getting replaced have white space characters or not.  Fo example, you have `PLGRD/|PLGRD\s` but just `PLAYLOT\s`

Reply:
Replacements are happening to `' '||TRIM(parcel_name)||' '`. The spaces get added to more easily identify ends of words. Examples where `PLGRD/` would get replaced include: `PS 192 PLGRD/JACOB SCHIFF` and `P.S.28 PLGRD/HIGHBRIDGE PARK`. Examples where `PLGRD\s` would get replaced include: `JHS 144 & L B BURNS PLGRD` and `S ST. CPORT/BURLING SLIP/PLGRD`